softdev.bib

```diff
@@ -2571,3 +2571,14 @@ @inproceedings{trudel13really
     month = Jul,
     doi = "10.1007/978-3-642-39038-8_20"
 }
+
+@inproceedings{Carr2017,
+  doi = {10.1145/3052973.3052983},
+  url = {https://doi.org/10.1145/3052973.3052983},
+  year = {2017},
+  month = Apr,
+  publisher = {{ACM}},
+  author = {Scott A. Carr and Mathias Payer},
+  title = {{DataShield: Configurable Data Confidentiality and Integrity}},
```

Comment:
We don't want the inner `{` as they stop bibtex doing auto-capitalisation. *But* we need it around `DataShield`. So we need either:

```
title = {{DataShield}: Configurable Data Confidentiality and Integrity}
```

or (as I tend to do it just so that I don't confuse myself):

```
title = "{DataShield}: Configurable Data Confidentiality and Integrity}",
```